dotcom-rendering/src/web/server/htmlTemplate.ts

```diff
@@ -286,15 +280,12 @@ https://workforus.theguardian.com/careers/product-engineering/
                 ${loadableConfigScripts.join('\n')}
                 ${priorityScriptTags.join('\n')}
                 <style class="webfont">${getFontsCss()}</style>
-                <style>${resets.resetCSS}${css}</style>
+                <style>${resets.resetCSS}</style>
+				${css}
 				<link rel="stylesheet" media="print" href="${CDN}static/frontend/css/print.css">
 			</head>
 
 			<body>
-				${skipToMainContent}
-				${skipToNavigation}
-				${skipToKeyEvents || ''}
-                <div id="react-root"></div>
```

Comment:
What was this div doing before? We weren't putting the `html` string into it, so it would be nice to know why it can now be removed (or if it was used beforehand)

Reply:
This div has moved into `Page`. It's used as the place where the client side React app we run is hooked into - in BootReact

Reply:
Ah yes thank you, I missed that one :)